### DISCUSSION TITLE
Getting exception void com.xceptance.xlt.api.engine.CustomData.setRunTime()' in the rerort.

### DISCUSSION THREAD

**Discussion by bhakarswayamavarapu** [Xceptance/XLT#397] [General]
Hi Team,
I was getting below error in the report after running load test of a T-Visit class with a measurement period of 5 min.
**java.lang.NoSuchMethodError: 'void com.xceptance.xlt.api.engine.CustomData.setRunTime()'**
![MicrosoftTeams-image (2)](https://github.com/Xceptance/XLT/assets/140174174/79483f0d-9da3-416c-bbdc-e23579c4193b)

Can you please suggest? Thanks!

**Comment by rschwietzke** [comment]
Please check the XLT 7.0 release notes and migrate your test suite properly from XLT 6 to 7.
https://docs.xceptance.com/xlt/release-notes/7_0_x/

**Reply by bhakarswayamavarapu** [reply]
Hi @rschwietzke , Thanks for the info,
Currently, I am using the latest XLT version (XLT 7.1.0) and did not used the XLT 6 previously.

Can you please suggest?

**Reply by rschwietzke** [reply]
But your test suite is XLT 6 based, so you have to change your code to allow it to work with XLT 7.

**Reply by rschwietzke** [reply]
Which test suite did you use? Where did you get it from?

**Reply by bhakarswayamavarapu** [reply]
Hi @rschwietzke ,
I was using the below test suit got from XLT github.
[**posters-advanced-loadtest-suite**](https://github.com/Xceptance/posters-advanced-loadtest-suite)
Thanks!

**Reply by rschwietzke** [reply]
This one is not XLT 7 yet. We are updating it at the moment together with a Posters shop update. Stay tuned.

**Reply by rschwietzke** [reply]
Give the latest version a try: https://github.com/Xceptance/posters-advanced-loadtest-suite

**Comment by rschwietzke** [comment]
Test suite is updated.

**Reply by bhakarswayamavarapu** [reply]
Thank you. Will let you know if any issues.
